qcore/timeseries.py

```diff
@@ -54,13 +56,15 @@ def bwfilter(data, dt, freq, band, match_powersb=True):
     #    x *= -1
     # freq *= exp(x * math.log(sqrt(2.0) - 1.0))
     nyq = 1.0 / (2.0 * dt)
+    highpass_shift = 0.8956803352330285
+    lowpass_shift = 1.1164697500474103
```

Comment:
Just to be annoying, aren't these better as module level constants?

Reply:
They're used in one function, so I don't think so